analytics/find-hds-react-version-usage.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# Set environment variable to local variable
+TOKEN=$GITHUB_TOKEN
+# Call to Github API to find all occurrences of "hds-react" under City-of-Helsinki org.
+curl -H "Accept: application/vnd.github+json"  -H "Authorization: token $TOKEN" "https://api.github.com/search/code?q=hds-react+in:file+filename:package.json+org:City-of-Helsinki&per_page=100" | jq '. | .total_count as $total_count | { total_count: $total_count, repositories: [.items[] | {name: .repository.name, url: .repository.html_url, package_url: .git_url, path: .path}]}' > hds-react-version-usage.json
+
+
+# count variable is used to access the correct object inside hds-react-version-usage.json array
+count=0;
+# Iterate through hds-react-version-usage.json file
+jq -c '.repositories[]' hds-react-version-usage.json | while read i; do
+
+    # Read the git url for package json
+    url=$(echo $i | jq -r '.package_url');
+    # Get the contents of package.json through the URL
+    content=$(curl -H "Authorization: token $TOKEN" ${url} | jq -r '.content' | base64 --decode)
+    # Read the version for hds-react
+    version=$(jq '.dependencies."hds-react"' <<< "${content}")
+    # If version is null, read it from devDependencies instead
+    [[ "$version" = "null" ]] && properVersion=$(jq '.devDependencies."hds-react"' <<< "${content}") || properVersion=$(jq '.dependencies."hds-react"' <<< "${content}");
+    # Insert the key value pair "version": "xx.xx" into the hds-react-version-usage.json with sponge
+    jq --argjson properVersion "$properVersion" --argjson count "$count" '.repositories[$count] += { version: $properVersion }' hds-react-version-usage.json | sponge hds-react-version-usage.json;
+    
+    ((count++))
+done
+# Remove package helsinki-design-system from the list
+jq 'del(.repositories[] | select(.name == "helsinki-design-system"))' hds-react-version-usage.json | sponge hds-react-version-usage.json
```

Comment:
It would be nice if the result would be in the same results folder as other results.
It would be also useful if the name would include date as other results.

Reply:
This script does not seem to filter out duplicates 🤔 
Maybe it does not need to if the subfolders might have different versions

Reply:
Yes, I thought it could be useful to see all the versions inside a project if it using multiple versions. Then we could even holla at them to fix it.